src/main/java/org/cp/confisual/plantuml/Visualiser.java

```diff
@@ -0,0 +1,38 @@
+package org.cp.confisual.plantuml;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import net.sourceforge.plantuml.SourceStringReader;
+import net.sourceforge.plantuml.security.SFile;
+import org.cp.confisual.nevisauth.Domain;
+import org.cp.confisual.nevisauth.NevisAuthConfigVisitor;
+
+public class Visualiser {
```

Comment:
Should rename the class to 'NevisAuthVisualiser' and keep it in 'nevisauth' package